doc/api/os.md

```diff
@@ -49,6 +49,18 @@ Returns an object containing commonly used operating system specific constants
 for error codes, process signals, and so on. The specific constants currently
 defined are described in [OS Constants](#os_os_constants_1).
 
+## os.constrainedmem()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {integer}
+
+The `os.constrainedmem()` method returns the an integer that represents the
```

Comment:
Maybe reword "the an integer that represents" as "an integer representing"

Reply:
I would suggest removing "an intger that represent the", make the text as parallel as possible to the wording of the `freemem()` docs. Also, I know this has to do with returning the right max memory that is available to docker containers, because I read the issues, but I don't think anyone could possibly guess this from the hand wavy "constrained" description here. I suggest the docs be much more clear on what it is. Using docker by name isn't a bad thing, and also adding the exact /sys/... property that it is being read and returned on Linux isn't bad, either, it allows people to googl around and find out more information without reading the uv source.

Reply:
Sure, happy to add. Thanks for the suggestions!